### DISCUSSION TITLE
About the Font size of note 关于笔记的字体大小

### DISCUSSION THREAD

**Discussion by llwhyll** [windingwind/zotero-better-notes#881] [Ideas]
The translation interface can edit the font size, I think the note interface also needs to add a custom font size function... Because the current font of the notes is really small, I am really uncomfortable to look at.翻译的界面可以编辑字体大小，我觉得笔记的界面也需要添加自定义字体大小的功能...因为笔记目前的字体实在是太小了，我看着真的不舒服。
[截屏2024-01-30 10.36.17.pdf](https://github.com/windingwind/zotero-better-notes/files/14096466/2024-01-30.10.36.17.pdf)

**Comment by windingwind** [comment]
这不是笔记（Note），这是批注（Annotation）。与本插件无关
